apple-app-site-association.json

```diff
@@ -36,6 +36,7 @@
           "NOT /click/*/aHR0cHM6Ly9hcHAuYWRqdXN0LmNv*",
           "NOT /users/auth/facebook/callback*",
           "NOT /identity-verification*",
+          "NOT /reset_password",
           "NOT *L2FydGljbGVz*",
```

Comment:
I think we probably also want /forgot and /change-password here and possibly others, working on seeing if I can find a list

Reply:
That would be great, I will add an item about this to the practice as well

Reply:
Here is it, please feel free to extend it: https://www.notion.so/artsy/Mobile-Practice-ecc07763bfd04a848c74107dde3ec6dc#863ed7100327448faffbe6c5ac7d783c

Reply:
That looks complete, asked the dev channel and it looks like those are the only other 2 👍

Reply:
Cool then, I will add them now and merge

Reply:
Here is it https://github.com/artsy/artsy-eigen-web-association/pull/37